common/buildings/ancient_magic_city_buildings.txt

```diff
@@ -1292,10 +1292,7 @@ guild_halls_01 = {
 		modifier = {
 			add = 500
 			scope:holder = {
-				OR = {
-					has_government = magocracy_government
-					has_government = republic_government
-				}
+				has_government = republic_government
```

Comment:
has_government = magocracy_government is missing in this OR and many other ORs in this file